src/java/org/apache/cassandra/index/sai/utils/AbortedOperationException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.index.sai.utils;
+
+
+/**
+ * This exception indicates that a request was aborted, normally because it was taking too much time.
+ *
+ * It is handled in a special way by the verb handlers and the request execute method: it is simply
+ * passed to the onAborted callback without logging any message. Therefore if any logging is required,
```

Comment:
I was going to ask if we could `{@link blah}` to where that happens, but then I realized I couldn't find it.

Reply:
As it happens, we shouldn't be using this exception anyway because it isn't handled correctly by the query path. I have changed `QueryContext` to throw a `QueryCancelledException` which is the same as is thrown by storage and handled correctly.

I am deleting this class.